### DISCUSSION TITLE
I finally have this installed, but no entities in dashboard

### DISCUSSION THREAD

**Discussion by limestar** [sebr/bhyve-home-assistant#87] [Q&A]
I know it is finally working based on looking at the logs showing login worked with orbit website.

However, I don't have any entities showing. Any debugging ideas as to why?

*I've installed and reinstalled this about 6 times and working on it over the past 3 days.
**installed on VM windows box.

**Comment by sebr** [comment]
Here's how you can enable debugging:

Step 1. Add the following to `configuration.yaml` and then restart Home Assistant.
```
logger:
  logs:
    custom_components.bhyve: debug
```

Step 2: Check Home Assistant logs

1. Open `home-assistant.log`
2. Identify lines which begin with:
```log
DEBUG (MainThread) [custom_components.bhyve] Devices:
DEBUG (MainThread) [custom_components.bhyve] Programs:
```

Copy these two lines and paste them below.

```log
DEBUG (MainThread) [custom_components.bhyve] Devices:
DEBUG (MainThread) [custom_components.bhyve] Programs:
```

**Comment by limestar** [comment]
Update. Fixed some config files that I believe were due to copy and pasting code.

When I add an entities card:
I have 4 options + the automation to select. 
sprinkler state, my watering program, if ON rain delay, if Smart Watering Program, and then the automation.

Should I be able to add status and run_time etc in there?

**Reply by limestar** [reply]
I believe I have this fully installed now. This integration was one of the main reasons I'm moving to Home Assistant from Smartthings. Thank you.

**Reply by sebr** [reply]
Everything seems to look okay in your debug logs - I have no reason to believe that the entities shouldn't be available in Home Assistant.

Did you resolve the issue and are you all set now?

**Reply by limestar** [reply]
Yes 100% resolved.
